pkg/cri/server/image_pull.go

```diff
@@ -375,6 +375,7 @@ func (c *criService) registryHosts(ctx context.Context, auth *runtime.AuthConfig
 				}
 			} else if isLocalHost(host) && u.Scheme == "http" {
 				// Skipping TLS verification for localhost
+				transport.TLSClientConfig = &tls.Config{}
```

Comment:
is it possible to init tls.Config in `newTransport` ?

Reply:
The TLSClientConfig structure will take up more space. Try not to apply for space when we are sure not to use it?
It is also good to apply here in advance if it is necessary to ensure that the other places are panic again.